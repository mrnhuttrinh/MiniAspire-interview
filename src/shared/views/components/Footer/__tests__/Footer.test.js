import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../Footer';

describe('Footer', () => {
  it('should render Footer Component', () => {
    const component = shallow(<Footer />);
  
    expect(component).toMatchSnapshot();
  });
});