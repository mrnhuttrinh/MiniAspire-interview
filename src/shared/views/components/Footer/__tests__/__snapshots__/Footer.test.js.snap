// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render Footer Component 1`] = `
<footer
  className="footer"
>
  A Footer
</footer>
`;
